test/dlc_tests/benchmark/ec2/pytorch/training/test_performance_pytorch_training.py

```diff
@@ -2,15 +2,40 @@
 
 import pytest
 
-from test.test_utils import CONTAINER_TESTS_PREFIX
+from test.test_utils import CONTAINER_TESTS_PREFIX, PT_GPU_PY3_BENCHMARK_IMAGENET_AMI, DEFAULT_REGION
 from test.test_utils.ec2 import execute_ec2_training_performance_test
 
 
 PT_PERFORMANCE_TRAINING_GPU_SYNTHETIC_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "benchmark", "run_pytorch_training_performance_gpu_synthetic")
+PT_PERFORMANCE_TRAINING_GPU_IMAGENET_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "benchmark", "run_pytorch_training_performance_gpu_imagenet")
 
 PT_EC2_GPU_SYNTHETIC_INSTANCE_TYPE = "p3.16xlarge"
+PT_EC2_GPU_IMAGENET_INSTANCE_TYPE = "p3dn.24xlarge"
 
 
 @pytest.mark.parametrize("ec2_instance_type", [PT_EC2_GPU_SYNTHETIC_INSTANCE_TYPE], indirect=True)
```

Comment:
This test is to be as a comparison to verify this env works fine with the test using common AMI